sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -219,14 +219,19 @@ def encrypt(self, algorithm, plaintext, **kwargs):
         Requires the keys/encrypt permission. This method encrypts only a single block of data, whose size depends on
         the key and encryption algorithm.
 
-        :param algorithm: encryption algorithm to use
+        :param algorithm: Encryption algorithm to use
         :type algorithm: :class:`~azure.keyvault.keys.crypto.EncryptionAlgorithm`
-        :param bytes plaintext: bytes to encrypt
-        :keyword bytes iv: initialization vector. Required for only AES-CBC(PAD) encryption.
-        :keyword bytes additional_authenticated_data: optional data that is authenticated but not encrypted. For use
+        :param bytes plaintext: Bytes to encrypt
+        :keyword bytes iv: Initialization vector. Required for only AES-CBC(PAD) encryption. If you pass your own IV,
+            make sure you use a cryptographically random, non-repeating IV. If omitted, an attempt will be made to
+            generate an IV via `os.urandom <https://docs.python.org/library/os.html#os.urandom>`_ for local
```

Comment:
Is the underscore after the closing backtick intentional?

Reply:
It is, for producing a hyperlink in our generated docs 🙂